### DISCUSSION TITLE
clinet quit unexpectedly

### DISCUSSION THREAD

**Discussion by kimiwanano** [rustdesk/rustdesk#5620] [General]
When I try to use the mac side to control the windows side(rustdesk v1.2.3), there are two problems:
The first issue is the same as this one: https://github.com/rustdesk/rustdesk/issues/732 
I achieve the connection through forced relay;
The second problem is that after my mac connects to the windows, the windows will crash after just a few seconds, and the error on the mac is as follows:
[2023-09-07T08:15:45Z INFO  librustdesk::client] Start text clipboard loop
INFO:TIS: Update cursor, system: false
INFO:TIS: Update cursor, system: false
INFO:TIS: enter
[2023-09-07T08:16:04Z ERROR librustdesk::client] Connection closed: Connection reset by peer (os error 54)

**Comment by rustdesk** [comment]
You need to tell more information about the windows, the mac info is useless.
